manifests/mcollective.pp

```diff
@@ -1,7 +1,7 @@
 define clamps::mcollective (
-  $user = $title,
-  $amqpass = 'password',
-  $amqserver = '$::server',
+  $user       = $title,
+  $amqpass    = 'password',
+  $amqservers = [$::servername],
```

Comment:
I ask mostly because `facter fqdn` on aws nodes is not always the same as their CN